.stylelintrc

```
@@ -0,0 +1,19 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "indentation": 4,
+    "max-empty-lines": 2,
+    "no-invalid-double-slash-comments": null,
+    "rule-empty-line-before": null,
+    "at-rule-empty-line-before": null,
+    "declaration-empty-line-before": null,
+    "selector-pseudo-element-colon-notation": "single",
+    "no-descending-specificity": null,
+    "selector-disallowed-list": [
+      "/^[.#]js-/",
+      {
+        "message": "js-* selectors can’t have visual styling. https://stackoverflow.design/product/guidelines/javascript/#javascript-classnames"
+      }
+    ],
```

nit: Drop trailing comma in json files

```suggestion
    ]
```